### PR TITLE
Docs/cli design doc

### PR DESCRIPTION
**Pull request by toszo**
#117

### CODE REVIEW

File: docs/design-docs/cli/cli.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    # ...
+```
+
+Many configuration files will be handled using `---` document separator. Like:
+
+```yaml
+kind: configuration/kubernetes
+# ...
+---
+kind: configutaion/kafka
```
Comment (erzetpe) [inline docs/design-docs/cli/cli.md]: Shouldn't be configuration?

File: docs/design-docs/cli/cli.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+8. plugable storage/vault for Epiphany state and Terraform state
+
+## Use cases
+
+### CLI deployments/management usage
+
+Create empty cluster:
+
+```bash
+> epiphany create cluster --name='epiphany-first-cluster'
```
Comment (erzetpe) [inline docs/design-docs/cli/cli.md]: Wouldn't it be better to use something shorter or more meaningful(eg epicli, epictl)?

File: docs/design-docs/cli/cli.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Same as many configurations can be enclosed in one file with `---` separator, configuration and infrastructure `yamls` should also be treated in that way.
+
+Example:
+
+```yaml
+kind: configuration/kubernetes
+# ...
+---
+kind: configutaion/kafka
```
Comment (erzetpe) [inline docs/design-docs/cli/cli.md]: Configuration?

File: docs/design-docs/cli/cli.md (hunk 1 above)
Comment (toszo) [reply docs/design-docs/cli/cli.md]: yes is should :)

File: docs/design-docs/cli/cli.md (hunk 2 above)
Comment (toszo) [reply docs/design-docs/cli/cli.md]: This is just a proposition - but `epictl` sounds great

Comment (erzetpe) [reply docs/design-docs/cli/cli.md]: Go for epictl. :) 👍

Comment (seriva) [reply docs/design-docs/cli/cli.md]: my vote for epicli since that can be said as epicly

**Pull request merged**